contracts/governance/AxelarServiceGovernance.sol

```diff
@@ -11,36 +11,43 @@ import { BaseMultisig } from './BaseMultisig.sol';
  * @dev This contract is part of the Axelar Governance system, it inherits the Interchain Governance contract
  * with added functionality to approve and execute multisig proposals.
  */
-contract AxelarServiceGovernance is InterchainGovernance, BaseMultisig, IAxelarServiceGovernance {
+contract AxelarServiceGovernance is InterchainGovernance, IAxelarServiceGovernance {
     enum ServiceGovernanceCommand {
         ScheduleTimeLockProposal,
         CancelTimeLockProposal,
         ApproveMultisigProposal,
         CancelMultisigApproval
     }
 
+    address public immutable interchainMultisig;
```

Comment:
since both this and the multisig isn't upgradable, can you make this storage variable and add a transferMultisig method so we can migrate multisig contract without having to go through governance